File: Recursion/add_twoNumber.py
def add_recursive(a, b):
    if b == 0:
        return a
    else:
        return add_recursive(a + 1, b - 1)

num1 = int(input())
num2 = int(input())
result = add_recursive(num1, num2)
print("Sum =", result)

File: Recursion/add_twoNumber.c
#include <stdio.h>

int add(int a, int b) {
    if (b == 0)
        return a;
    else
        return add(a + 1, b - 1);
}

int main() {
    int num1, num2;
    printf("Enter two integers: ");
    scanf("%d %d", &num1, &num2);
    int result = add(num1, num2);
    printf("Sum = %d\n", result);
    return 0;
}